receiver/githubactionsreceiver/log_event_handling.go

```diff
@@ -23,8 +23,29 @@ import (
 
 func eventToLogs(event interface{}, config *Config, ghClient *github.Client, logger *zap.Logger, withTraceInfo bool) (*plog.Logs, error) {
 	if e, ok := event.(*github.WorkflowRunEvent); ok {
```

Comment:
(just my eyes, not asking for a change 😁)

_twitch_ - if this got inverted the whole function could be outdented one level!